deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -44,6 +44,7 @@ def GenerateConfig(context):
 
     FORSETI_HOME = '$USER_HOME/forseti-security'
     POLICY_LIBRARY_HOME = '$USER_HOME/policy-library'
+    POLICY_LIBRARY_SYNC_ENABLED = 'false'
```

Comment:
The new sync feature is only being supported from within Terraform. Installations using the deprecated method will have this feature disabled.